scalableminds/webknossos
# Add Volume Tasks

### URL of deployed dev instance (used for testing):
- https://volumetasks.webknossos.xyz

### Steps to test:
- create new task type, select volume as tracing type
- create tasks with this task type (try bulk + form-based. file-upload is not yet supported)
- request tasks, trace, finish
- download project, you should get zip with nmls and data zips
- task type tracing type should be immutable (since tasks once created have their initialized tracing bases)
- viewing the project/tasktype as compound annotation should fail (since merging volume tracings is not yet defined)
- skeleton task handling should be unaffected

### Issues:
- fixes #3668 

------
- [x] Ready for review [Backend]
- [x] Ready for review [Frontend]
